### PR TITLE
Points counting

### LINKED ISSUE #38: 1. Counting points for unfinished games
Given that we already have data from input files
- we should check if the game is not finished (if the number of frames from the input is less than 10)
- we should check if all the players have the same number of frames?

### LINKED ISSUE #39: 2. Counting points for finished games
Given that we already have data from input files: 

The same checks as in 1. Counting points for unfinished games:
- we should check if the game is finished (if number of frames is equal to 10)
- we should check if every player has the same number of frames\
 Additionally:
- we should check if extra balls occured

### LINKED ISSUE #54: 9. Made a function that convert vector<Frames> (chars) into size_t
Made a function that convert vector<Frames> (chars) into size_t and put this into counting methods to count point correctly, straight from vector<Frames>, fix tests.

### PR DESCRIPTION
**Pull request by Marcinati**

### CODE REVIEW

File: bowling/Game.cpp [hunk 1]
```diff
@@ -77,6 +77,58 @@ Game::Status Game::getGameStatus() const {
...
+    return totalOnlyStrikePoints;
+}
+
+size_t Game::countOnlySpareFrames(std::vector<Frame>& rolls) {
+    size_t totalOnlySparePoints = 0;
+    for (size_t i = 0; i < rolls.size(); i++) {
+        if (isSpare(rolls[i].getSecondRoll())) {
+            totalOnlySparePoints += 10;
+            if ((i + 1) != rolls.size()) {
+                if (isStrike(rolls.at(i + 1).getFirstRoll()) || isSpare(rolls.at(i + 1).getSecondRoll())) {
```
Comment (WojtekMs) [inline bowling/Game.cpp]: I think there is a mistake here. If we have a spare in current frame, points for this frame are:
10 + points for the next roll
If there is a spare in the next frame, we still only add points scored for the first roll.
I think that the condition should be just:
`isStrike(rolls.at(i + 1).getFirstRoll()) `

**Review by WojtekMs — COMMENTED** [review]
Good code! :)

File: bowling/Game.cpp (hunk 1 above)
Comment (Marcinati) [reply bowling/Game.cpp]: You are right. I've changed this, thanks.

**Review by WojtekMs — APPROVED** [review]
All good :)

**Pull request merged**